pages/common/convmv.md

```diff
@@ -0,0 +1,11 @@
+# convmv
+
+> Converts filenames from one encoding to another
+
+- convert encoding(dry run)
```

Comment:
Space added. Without `--notest` the command just shows what it wants to do, since the operation could be destructive.